package.json

```diff
@@ -30,13 +28,21 @@
   "devDependencies": {
     "babel": "^5.5.8",
     "babel-eslint": "^3.1.15",
+    "babel-loader": "^5.1.4",
     "dependency-check": "^2.4.0",
     "eslint": "^0.23.0",
-    "faucet": "0.0.1",
-    "istanbul": "^0.3.15",
+    "isparta-loader": "^0.2.0",
+    "jasmine": "^2.3.1",
+    "karma": "^0.12.36",
+    "karma-chrome-launcher": "^0.1.12",
+    "karma-coverage": "^0.4.2",
+    "karma-jasmine": "^0.3.5",
+    "karma-nyan-reporter": "0.0.60",
+    "karma-webpack": "^1.5.1",
+    "eslint": "^0.23.0",
     "nsp": "^1.0.1",
     "precommit-hook": "^2.0.1",
     "rimraf": "^2.4.0",
-    "tape": "^4.0.0"
+    "webpack": "^1.9.11"
```

Comment:
That sounds great, but let's get this merged first so you can do that in a separate PR. =)